tasks/ubuntu.yml

```diff
@@ -1,10 +1,20 @@
 ---
 
+- name: Ubuntu | Add an Apt signing key used for podman repo
+  apt_key:
+    url: "{{ podman_apt_pub_key }}"
+    state: present
+
 - name: Ubuntu | Add podman repository
   become: true
   become_user: root
   apt_repository:
     repo: "{{ podman_apt_repo }}"
+    filename: podman
+
+- name: Ubuntu | Upgrade System before install
+  apt:
+    upgrade: dist
```

Comment:
With the above comment we should then do:

```
when: repository_added.changed
```

Otherwise we'll do a system upgrade every time the user runs a playbook.